pkg/kubelet/kubelet_pods.go

```diff
@@ -559,7 +559,7 @@ func (kl *Kubelet) getServiceEnvVarMap(ns string, enableServiceLinks bool) (map[
 		// from the master service namespace, even if enableServiceLinks is false.
 		// We also add environment variables for other services in the same
 		// namespace, if enableServiceLinks is true.
-		if service.Namespace == kl.masterServiceNamespace && masterServices.Has(serviceName) {
+		if service.Namespace == metav1.NamespaceDefault && masterServices.Has(serviceName) {
```

Comment:
nit: you could adjust the comment above accordingly(?)

Reply:
Thanks @marquiz Good catch! Updated.